include/net/net_pkt.h

```diff
@@ -2367,9 +2367,9 @@ int net_pkt_read_be32_new(struct net_pkt *pkt, u32_t *data);
 /**
  * @brief Write data into a net_pkt
  *
- * Note: net_pkt's cursor should be properly initialized and,
- *       eventually, properly positioned using net_pkt_skip_read/write.
- *       Cursor will be updated according to parameters.
+ * Note: net_pkt's cursor should be properly initialized and, if needed,
+ *       positioned using net_pkt_skip_read/write.
```

Comment:
I'm almost sure that this should be changed to "positioned using net_pkt_skip", but would like to get confirmation too, @tbursztyka

Reply:
skip/read/write are valid here. Basically whatever updates the cursor is valid. (you can even set/unset overwrite in between etc...)

Reply:
> skip/read/write are valid here. Basically whatever updates the cursor is valid. (you can even set/unset overwrite in between etc...)

Oh, I finally got what is meant there. And that's what I meant in the commit description - deemphasize aux stuff in the description, because it only confusing. The text talks about *positioning* cursor, and positioning happens via net_pkt_skip(). Yes, `*_read()`/`*_write()` update cursor position, but that fact should be described in `*_read()`/`*_write()` docstrings, not everywhere we talk about a cursor. So, going to use "positioned using net_pkt_skip." everywhere.